src/Core/LeanCode.Components.Startup/IHostBuilderExtensions.cs

```diff
@@ -32,6 +32,24 @@ public static IHostBuilder AddAppConfigurationFromAzureKeyVault(this IHostBuilde
             });
         }
 
+        public static IHostBuilder AddAppConfigurationFromAzureKeyVault(
+            this IHostBuilder builder,
+            string vaultKeyOverride,
```

Comment:
If this is mandatory, it is no longer an `override` IMO.

Reply:
It's not mandatory if constructor with default values exists.